ui/k-toolbar/main.kytos

```diff
@@ -79,10 +80,13 @@ module.exports = {
        dataType: "json",
        contentType: "application/json; charset=utf-8",
        success: function(data, textStatus){
-           console.log('Circuit with id '+data.circuit_id+" was creaded.")
+           console.log('Circuit with id '+data.circuit_id+" was created.")
+           alert('Circuit with id '+data.circuit_id+" was created.")
+
```

Comment:
Suggestions: add spaces to increase readability; use variable to avoid repeating code.

```suggestion
           msg = 'Circuit with id ' + data.circuit_id + " was created."
           console.log(msg)
           alert(msg)
```

Reply:
resolved with commit f1f509c4b09e995b0c89e69aaf0c759c2634e6cd